src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,136 @@
+package wolox.training.models;
+
+import com.sun.istack.NotNull;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
It is recommended to use SEQUENCE as a strategy because Postgres uses a global incremental id shared by all tables :)

Reply:
The problem with that shared sequence is that it reaches a point that the sequential integrity, usually I use GenerationType.IDENTITY that directly the DB takes care of that but what happened now was copy and paste because it does not look well haha